services/start-services-only.sh

```diff
@@ -7,5 +7,5 @@ set -eo pipefail
 
 SERVICES_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# call integration.sh, show a message, then sleep (instead of executing a test executable)
-"$SERVICES_DIR/integration.sh" bash -c 'printf "(This will hang, Control+C to close.)\nNow you can manually curl them or start an integration test executable manually with e.g. \n(first cd to a service dir for correct working directory)\n  cd services/brig && ../../dist/brig-integration -s brig.integration.yaml -i ../integration.yaml\n" && sleep 1000000'
+# call run-services, show a message, then sleep (instead of executing a test executable)
+"/usr/bin/python3" "$SERVICES_DIR/run-services" bash -c 'printf "(This will hang, Control+C to close.)\nNow you can manually curl them or start an integration test executable manually with e.g. \n(first cd to a service dir for correct working directory)\n  cd services/brig && ../../dist/brig-integration -s brig.integration.yaml -i ../integration.yaml\n" && sleep 1000000'
```

Comment:
Depending on whether Nix's python should be used or not, this needs to be changed as well.